conda-forge.yml

```diff
@@ -1,6 +1,14 @@
+bot:
+  automerge: true
```

Comment:
I am not a friend of the bot automerging - cleaning up when things go wrong feels like a big hassle.

Reply:
Okay we can turn this off.  I've generally not had any issues myself as I feel that nowadays conda-forge CI is pretty good at catching most issues, but I'll leave this to your discretion.